test/cmd/admin.sh

```diff
@@ -155,6 +155,30 @@ os::cmd::expect_success 'oadm policy add-cluster-role-to-group cluster-admin sys
 os::cmd::expect_success 'oadm policy remove-cluster-role-from-group cluster-admin system:unauthenticated'
 os::cmd::expect_success 'oadm policy add-cluster-role-to-user cluster-admin system:no-user'
 os::cmd::expect_success 'oadm policy remove-cluster-role-from-user cluster-admin system:no-user'
+# test the rolebinding and clusterrolebinding name flag
+os::cmd::expect_success 'oadm policy add-role-to-user admin -z fake-sa1 -N rb1'
+os::cmd::expect_success 'oadm policy add-role-to-user admin -z fake-sa2 -N rb2'
+os::cmd::expect_success 'oadm policy add-role-to-user admin -z fake-sa3 -N rb2'
+os::cmd::expect_success_and_text 'oc get rolebinding/rb1' 'rb1       /admin                        fake-sa1'
```

Comment:
It is, when the paths are comma-delimited. See [the doc](https://kubernetes.io/docs/user-guide/jsonpath/)